eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriterTest.java

```diff
@@ -149,11 +150,15 @@ public void checkReference() throws Exception {
 
     @Test
     public void checkMetadataSection() throws Exception {
+        FileWriter fileWriter = new FileWriter(temporaryFolder.newFile(getContigsPath(REFERENCE_ASSEMBLY)));
+        fileWriter.write("CM0001.1");
+        fileWriter.close();
+
         File output = temporaryFolder.newFile();
         assertWriteVcf(output, buildVariant(CHR_1, 1000, "C", "A", SNP_SEQUENCE_ONTOLOGY, STUDY_1));
 
         List<String> metadataLines = grepFile(output, "^##.*");
-        assertEquals(9, metadataLines.size());
+        assertEquals(10, metadataLines.size());
```

Comment:
Maybe something a bit more specific like `##contig=<ID=CM0001.1>`? I think that is exactly what has to be written, otherwise it could be a badly-formed meta entry.